frontend/src/components/AdminUserReviewTable.js

```diff
@@ -66,7 +66,7 @@ function EmptyReviews() {
   return <h3>No reviews yet.</h3>;
 }
 
-function AdminReviewTable(props) {
+function AdminUserReviewTable(props) {
```

Comment:
I know that we were talking about making a review table for admins and a review table for users that just made the delete button invisible. Is that what this is because the naming is a bit confusing.

Reply:
both the review table within the admin and user page can delete comments, it's the reviews in the single hike page that shouldn't be deleted. So this is why Admin and Users use the same component but the SinglePage.js uses a different one.